lambdas/data-migration1/src/index.ts

```diff
@@ -0,0 +1,134 @@
+import AWS from 'aws-sdk';
+import Knex from 'knex';
+
+import DynamoDbSearchQueue from '@cumulus/aws-client/DynamoDbSearchQueue';
+import { connection } from '@cumulus/db';
+import Logger from '@cumulus/logger';
+
+const {
+  Manager,
+} = require('@cumulus/api/models');
+const schemas = require('@cumulus/api/models/schemas');
+
+const logger = new Logger({ sender: '@cumulus/data-migration' });
+
+export interface HandlerEvent {
```

Comment:
Why is this needed over the default NodeJS.ProcessEnv?      This definition safely conforms to NodeJS.ProcessEnv without providing additional information.

Reply:
Particularly as `migrateCollections` takes NodeJS.ProcessEnv

Reply:
There is probably no reason. This was copypasta. However, is it not helpful to specify that the handler event may contain a key `env` and that key should conform to NodeJS.ProcessEnv

Reply:
This seems fine, it's saying that an `env` can be passed in through the event, and using `NodeJS.ProcessEnv` seems the right type.